### ISSUE TITLE
Popout window freezes once app is minimized 

### ISSUE THREAD

**Issue by cutterchop** [ThaUnknown/miru#151]
**Describe the bug**
Pressing on the popout window on the app's player, it successfully pops out a mini version which you can drag outside of the app and roam the desktop with, problem is player automatically stops playing once you minimize the app.

**To Reproduce**
1. Open an anime episode
2. Click on Popout window 
3. Minimize the app
4. Popout player gets stuck

**Expected behavior**
I expect the mini player to continue playing on background usage when the app is minimized.

**Screenshots**
Sent a gif for a clear example of what is happening, in the middle of the video I even show case when I press on the video player pause button it does not respond

https://user-images.githubusercontent.com/31880937/163712797-5a2fa7c0-4971-4617-baf1-3df545129430.mp4

**Desktop (please complete the following information):**
 - OS: Windows 11 21H2
 - App version 1.6.0

**Additional context**

**Comment by ThaUnknown** [comment]
due to how browsers work, this isn't fixable, likely windows 11 ignores `backgroundThrottling: false`
TLDR: when browsers aren't visible they optimise themselves, reducing various update rates, this includes video playback and canvas rendering, which is how this PiP works

**Comment by ThaUnknown** [comment]
duplicate of #148 ?
